tests/framework/clients/rancher/v1/client.go

```diff
@@ -215,6 +232,73 @@ func (c *SteveClient) Delete(container *SteveAPIObject) error {
 	return c.apiClient.Ops.DoResourceDelete(c.steveType, &container.Resource)
 }
 
+func (c *NamespacedSteveClient) Create(container any) (*SteveAPIObject, error) {
```

Comment:
I want to make sure I understand the purpose of the NamespacedSteveClient. This really more for the usage of the `List` command set correct? Because it looks like the other commands don't change. Next question this for us to have the ability to list resources from a specific namespace? What would be the need to do that? Couldn't we just use the url.Values to set the namespace? I might be wrong just thought I'd ask.

Reply:
Ok cool totally understand. One question, maybe I just missed it, but I don't see how the namespace field for the namespace client is being used for the other CRUD operations.

Reply:
You're right, I wasn't using any of these other operations in my tests so I skipped over actually implementing it, will fix.